app.go

```diff
@@ -116,6 +116,12 @@ type App struct {
 	latestGroup *Group
 	// TLS handler
 	tlsHandler *TLSHandler
+	// check added routes of sub-apps
+	subAppsRoutesAdded bool
+	// Returns parent app if app was mounted
+	parentApp *App
```

Comment:
property is only set in the whole code, never read out 

do we really need this then?

Reply:
parentApp property is still not really in use